pkg/opcua/driver/client_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This application needs physical devices.
+// Please edit by demand for testing.
+
+package driver
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRead(t *testing.T) {
+
+	c := &OPCUAConfig{URL: "opc.tcp://localhost:4840",
+		/*
+		User:           "testuser",
+		Passwordfile:   "/home/wei/ca/pass",
+		SecurityPolicy: "None",
+		SecurityMode:   "None",
+		*/
+		/*
+			SecurityPolicy: "Basic256Sha256",
+			SecurityMode:   "Sign",
+			Certfile:       "/home/wei/ca/clientcert.pem",
+			Keyfile:        "/home/wei/ca/clientkey.pem",
+			RemoteCertfile: "/home/wei/ca/servercert.pem",
+		*/
```

Comment:
We can remove it if unnecessary

Reply:
Sorry, I'll refine the test. There're several test cases in one test function.